autoscale-docker-swarm-shared/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015-2017 Hewlett Packard Enterprise Development LP.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.github.autoscaler</groupId>
+        <artifactId>autoscale-aggregator</artifactId>
+        <version>1.0.1-SNAPSHOT</version>
+    </parent>
+    
+    <artifactId>autoscale-docker-swarm-shared</artifactId>
+    <groupId>com.github.autoscaler</groupId>
+    
+    <properties>
+        <skipITs>true</skipITs>
```

Comment:
Yes this is a standard property acted upon by the failsafe plugin.  So the container starts up and checks that it starts up correctly, but was the JIRA for integration tests isn't completed we cannot enable them.  The integration tests are still useful on a developers machines for now until this work is done, but they just have to run the IT files by hand.  PS. The above doesn't prevent unit tests, they still run.